src/services/validators/validate-cron.js

```diff
@@ -40,7 +40,9 @@ const isValidFraction = (fraction, x, y) => {
 
     /* istanbul ignore next */
     return (
-        (isWildcard(components[0]) || isValidNumber(components[0], x, y)) &&
+        (isWildcard(components[0]) ||
+            isValidNumber(components[0], x, y) ||
+            isValidNumberRange(components[0], x, y)) &&
         isValidNumber(components[1], x, y)
```

Comment:
This is a very hard to read section btw. Not your doing since this whole file has logic like that, but it's definitely in need of us cleaning it up.

Reply:
Agreed. Out of scope of this PR